05-lists-and-dictionaries/capitals.py

```diff
@@ -0,0 +1,14 @@
+# Copyright (c) 2016 Ami . All rights reserved
+
+capitals_dict = {
+    "Alabama": "Montgomery",
+    "Alaska": "Juneau",
+    "Arizona": "Phoenix",
+    "Arkansas": "Little Rock",
+    "California": "Sacramento",
+    "Colorado": "Denver",
+    "Connecticut": "Hartford",
+    "Delaware": "Dover",
+    "Florida": "Tallahassee",
+    "Georgia": "Atlanta"
```

Comment:
Add a comma `,` after `"Atlanta"` (in dictionaries with multiple lines, we add a comma at the end of each line).